RCSwitch.cpp

```diff
@@ -583,6 +596,9 @@ void RCSwitch::enableReceive() {
   if (this->nReceiverInterrupt != -1) {
     RCSwitch::nReceivedValue = 0;
     RCSwitch::nReceivedBitlength = 0;
+    #if defined(CORE_TEENSY)
+      pinMode(this->nReceiverInterrupt, INPUT);
+    #endif
     attachInterrupt(this->nReceiverInterrupt, handleInterrupt, CHANGE);
```

Comment:
This change is unrelated to this pull request -- please either remove it here and put it into a separate pull request, or exaplin why it must be mixed with this pull request.